### DISCUSSION TITLE
Make logging of OpenVPN and Transmission user-adjustable

### DISCUSSION THREAD

**Discussion by phesster** [haugene/docker-transmission-openvpn#2563] [Ideas]
I have some simple adjustments to "openvpn/modify-openvpn-config.sh" and "transmission/start.sh" that can make the verbosity of logging user-adjustable.  I don't know if this is something anyone else wants, but I find that these changes are working fine for my purposes.

--phess

[user-adjustable-logging.patch](https://github.com/haugene/docker-transmission-openvpn/files/11083699/user-adjustable-logging.patch)

**Comment by pkishino** [comment]
please submit as a PR for evaluation,thanks

**Comment by phesster** [comment]
PR submitted.  My apologies if I've missed something.  I haven't submitted anything to an Open Source project since the days of submitting unified text diffs.  :)

--phess

**Reply by pkishino** [reply]
Thanks, looks good. Merged into dev branch
